### DISCUSSION TITLE
how can i setup renovate to disable updating FROM in Dockerfiles?

### DISCUSSION THREAD

**Discussion by raz3k** [renovatebot/renovate#22027] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

ability to update using regexmanagers but be able to disable FROM updates in the Dockerfiles.

### What you tried so far.

I did not find this option documented.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by raz3k** [comment]
I want the ability to toggle this on and off in Dockerfiles

**Reply by rarkins** [reply]
To disable all Dockerfile updating write a package rule with matchManagers=dockerfile, enabled=false. You can user other package rules match rules if you want to narrow it further

**Reply by raz3k** [reply]
like this?
```
{
  "matchManagers": [
    "dockerfile"
  ],
  "packageRules": [
    {
      "matchPaths": [
        "Dockerfile.ignore-renovate"
      ],
      "ignorePaths": [
        "Dockerfile.ignore-renovate"
      ],
      "enabled": false
    }
  ]
}
```

**Reply by raz3k** [reply]
or like this ?
```
{
  "packageRules": [
    {
      "matchPaths": [
        "Dockerfile.ignore-renovate"
      ],
      "matchManagers": [
        "dockerfile"
      ],
      "ignorePaths": [
        "Dockerfile.ignore-renovate"
      ],
      "enabled": false
    }
  ]
}
```

**Reply by rarkins** [reply, marked as answer]
```
{
  "packageRules": [
    {
      "matchFiles": [
        "Dockerfile.ignore-renovate"
      ],
      "matchManagers": [
        "dockerfile"
      ],
      "enabled": false
    }
  ]
}
```
